Formula/libarchive.rb

```diff
@@ -31,9 +36,15 @@ class Libarchive < Formula
   uses_from_macos "expat"
   uses_from_macos "zlib"
 
+  # fix pkg-config file, remove in next release
+  patch do
+    url "https://github.com/libarchive/libarchive/commit/1f35c466aaa9444335a1b854b0b7223b0d2346c2.patch?full_index=1"
+    sha256 "c3a8b7b6324f95bbcb24a506e07a58fdac3e93d32d7fc67d5d20dd4ba63a3d49"
+  end
```

Comment:
Note that this only works on Linux, which provides library in glibc.

On macOS, `LIBICONV` is defined to `-liconv`. Still need another upstream fix.